java-frontend/src/test/java/org/sonar/java/cfg/CFGTest.java

```diff
@@ -518,12 +519,14 @@ public void switch_statement() {
       "void fun(int foo) { int a; switch(foo) { case 1: System.out.println(bar);case 2: System.out.println(qix);break; default: System.out.println(baz);} }");
     CFGChecker cfgChecker = checker(
       block(
+        element(Kind.INT_LITERAL, "1"),
```

Comment:
could be great to modify this test to see appear not only literals but also identifier.